src/main/java/persistence/sql/dml/clause/ChainingWhereClauseList.java

```diff
@@ -0,0 +1,37 @@
+package persistence.sql.dml.clause;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Spliterator;
+import java.util.stream.Stream;
+
+public class ChainingWhereClauseList implements Iterable<ChainingLogicalOperatorStandardWhereClause> {
```

Comment:
```suggestion
public class ChainingWhereClauses implements Iterable<ChainingLogicalOperatorStandardWhereClause> {
```

List 보단 복수형을 사용해주는 것이 더욱 좋습니다 :)